llvm/lib/Target/AMDGPU/SIPeepholeSDWA.cpp

```diff
@@ -1105,6 +1105,7 @@ void SIPeepholeSDWA::convertVcndmaskToVOP2(MachineInstr &MI,
                        .add(*TII->getNamedOperand(MI, AMDGPU::OpName::src0))
                        .add(*TII->getNamedOperand(MI, AMDGPU::OpName::src1))
                        .setMIFlags(MI.getFlags());
+  TII->fixImplicitOperands(*Converted);
```

Comment:
It just occurred to me that this shouldn't be introducing the VOP2 form. It should directly go to SDWA or do nothing

Reply:
Right, but I think the same problem exists for the other instructions that are handled in a similar way, i.e. `V_SUB_CO_U32_e64` and `V_ADD_CO_U32_e64`. It's not necessarily wrong, but sub-optimal. I can try to get rid of this, but I suggest to bring in this bug fix first.